app/uk/gov/hmrc/plasticpackagingtax/registration/views/partials/amendment/organisationDetails.scala.html

```diff
@@ -60,7 +60,7 @@
             viewUtils.summaryListRow(organisationNameMessageKey, registration.organisationDetails.businessName, None),
             viewUtils.summaryListRow("amend.pptRef.label", pptReferenceNumber, None),
             viewUtils.summaryListRow("amend.taxStartDate.label", viewUtils.displayDate(registration.liabilityDetails.startDate), None),
-            viewUtils.summaryListRow("amend.dateOfRegistration.label", Some(viewUtils.displayDate(registration.dateOfRegistration)), None),
+            viewUtils.summaryListRow("amend.dateOfRegistration.label", viewUtils.displayLocalDate(registration.dateOfRegistration), None),
```

Comment:
Are we ok if this returns None?

Reply:
Yeah, the ViewUtils thing will just omit output in that case.